### PR TITLE
Show logged in user status and public name

### PR DESCRIPTION
**Pull request by cquiroz**
This PR adds some improvements to the UI in particular when showing the public display name.
It also makes it more robust against errors renewing the ticket.

### CODE REVIEW

File: common/src/main/scala/explore/components/ConnectionsStatus.scala [hunk 1]
```diff
@@ -9,54 +9,52 @@ import crystal.Error
...
 
 object ConnectionsStatus {
   type Props = ConnectionsStatus
 
-  private def renderStatus(name: String)(status: Pot[StreamingClientStatus]): VdomNode = {
-    val (message, clazz) = status match {
-      case Error(t)     => (t.getMessage, ConnectionError)
-      case Pending(_)   => ("Mounting...", ConnectionWarning)
+  private def renderStatus(name: String)(status: Pot[StreamingClientStatus]): VdomElement = {
+    val (message, clazz, show) = status match {
```
Comment (rpiaggio) [inline common/src/main/scala/explore/components/ConnectionsStatus.scala:29]: Nice, thank you.

File: common/src/main/scala/explore/AppMain.scala [hunk 2]
```diff
@@ -79,7 +79,11 @@ trait AppMain extends IOApp {
     def repeatTokenRefresh(ssoURI: Uri, expiration: Instant, v: View[UserVault]): IO[Unit] =
       SSOClient
         .refreshToken[IO](ssoURI, expiration, v.set, IO.fromFuture)
-        .flatMap(u => repeatTokenRefresh(ssoURI, u.expiration, v))
+        .attempt
+        .flatMap {
+          case Right(u) => repeatTokenRefresh(ssoURI, u.map(_.expiration).getOrElse(expiration), v)
+          case Left(_)  => repeatTokenRefresh(ssoURI, expiration, v)
```
Comment (rpiaggio) [inline common/src/main/scala/explore/AppMain.scala:85]: Is the `Left` case when there's no longer a valid token?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/AppMain.scala (hunk 2 above)
Comment (cquiroz) [reply common/src/main/scala/explore/AppMain.scala:85]: We need to differentiate (in another PR) when there is a regular failure which we are ignoring here and when the user was kicked out of SSO for some reason
I'll handle that case in another PR

**Pull request merged**